public/css/ctrlo-bootstrap.css

```diff
@@ -1234,3 +1234,45 @@ small.search_term {
 
 #reveal_password:checked ~ .reveal_area {
   display: block; }
+
+/* disclosure widget */
+.expandable {
+  display: none; }
+
+.trigger[aria-expanded="true"] + .expandable {
+  display: block; }
+
+/* Person contact details popover */
+.person.contact-details.expandable {
+  max-width: 20em;
+  word-break: break-all;
+  border: 1px solid #036ee9;
+  border-radius: 6px;
+  -webkit-box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+  background-color: #fff;
+  padding: .5em;
+  margin: .5em;
+  z-index: 5;
+  position: absolute; }
+
+/* Column RAG */
+.rag {
+  padding: .2em;
+  border-left-style: solid;
+  border-left-width: 1.5em; }
+
+.rag.undefined {
+  border-left-color: #a09fa0; }
+
+.rag.negative {
+  border-left-color: #800000; }
+
+.rag.average {
+  border-left-color: #eda400; }
+
+.rag.positive {
+  border-left-color: #008400; }
+
+.rag.unexpected {
+  border-left-color: #891e88; }
```

Comment:
Do these colours have to be used for accessibility? It would be nice to use the old colours if possible, as these align with the rest of the site's colour scheme/branding.